README.md

```diff
@@ -1,68 +1,69 @@
+|Contributing| [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com) [![GitHub issues by-label](https://img.shields.io/github/issues/aiidateam/aiida-core/good%20first%20issue)](https://github.com/aiidateam/aiida-core/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)  [Contributor Guide](https://github.com/aiidateam/aiida-core/wiki)|
+
+
+## Features
+
+ -   **Workflows:** Write complex, auto-documenting workflows in
+     python, linked to arbitrary executables on local and remote
+     computers. The event-based workflow engine supports tens of
+     thousands of processes per hour with full checkpointing.
+ -   **Data provenance:** Automatically track inputs, outputs & metadata
+     of all calculations in a provenance graph for full
+     reproducibility. Perform fast queries on graphs containing
+     millions of nodes.
+ -   **HPC interface:** Move your calculations to a different computer
+     by changing one line of code. AiiDA is compatible with schedulers
+     like [SLURM](https://slurm.schedmd.com), [PBS
+     Pro](https://www.pbspro.org/),
+     [torque](http://www.adaptivecomputing.com/products/torque/),
+     [SGE](http://gridscheduler.sourceforge.net/) or
+     [LSF](https://www.ibm.com/support/knowledgecenter/SSETD4/product_welcome_platform_lsf.html)
+     out of the box.
+ -   **Plugin interface:** Extend AiiDA with [plugins](https://aiidateam.github.io/aiida-registry/) for new simulation codes (input generation & parsing), data types, schedulers, transport modes and more.
+ -   **Open Science:** Export subsets of your provenance graph and share them with peers or make them available online for everyone
+     on the [Materials Cloud](https://www.materialscloud.org).
+ -   **Open source:** AiiDA is released under the [MIT open source license](https://github.com/aiidateam/aiida-core/blob/develop/LICENSE.txt).
+
+## Installation
+
+Please see AiiDA's [documentation](https://aiida-core.readthedocs.io/en/latest/).
+
+## How to cite
+
+If you use AiiDA in your research, please consider citing the AiiDA paper:
 
 > Giovanni Pizzi, Andrea Cepellotti, Riccardo Sabatini, Nicola Marzari,
 > and Boris Kozinsky, *AiiDA: automated interactive infrastructure and
 > database for computational science*, Comp. Mat. Sci 111, 218-230
 > (2016); <https://doi.org/10.1016/j.commatsci.2015.09.013>;
 > <http://www.aiida.net>.
 
-License
--------
+## License
+
+AiiDA is distributed under the MIT open source license (see [`LICENSE.txt`](LICENSE.txt)).
 
-The terms of the AiiDA license can be found in the LICENSE.txt file.
+For a list of other open source components included in AiiDA, see the
+file [`open_source_licenses.txt`](open_source_licenses.txt).
```

Comment:
This file is outdated and there are references to files that no longer exist. We should really update it if we're going to keep it and link it

Reply:
Thanks, I'll have a look